### ISSUE TITLE
ejabberd 19.09.56 - Documentation

### ISSUE THREAD

**Issue by maniskec** [processone/ejabberd#3082]
Hello team,

I am new to ejabberd , I have cloned source code from git and i have followed the instructions mentioned in the below link
https://docs.ejabberd.im/admin/installation/#install-from-source-code
I am able to start ejabberdctl service and i am able to use pidgin as my client. 

Now ,  I have problem in changing Database configuration and i want to use mysql to log all activities. 

More over , I cannot able to enable Rest API and cannot able to generate oauth token.

There is no documentation available for ejabberd 19.09.56. 

Though I have modified ejabberd.yml file
    **db_type: mysql
    db_server: 10.20.200.57
    db_port: 3306 # the default
    db_database: ejabberd
    db_username: mani
    db_password: "Iopex@123"
    db_pool_size: 5**

I am getting this error:
**Failed to start ejabberd application: Invalid value of option modules->mod_mam->db_type: unsupported database: mysql**

Anybody, Help me out to get documentation for ejabberd 19.09.56.

Thanks in advance.

**Comment by licaon-kter** [comment]
Use the latest documentation if you compile from git HEAD. Which part you can't find on docs exactly? 

Attach a gist of your config.

`db_type: sql` and not "mysql"

**Comment by maniskec** [comment]
Hello @licaon-kter ,
Still i am not able to connect mysql, I have changed ejabberd.yml as db_type: sql.

ejabberd is still using mnesia database to log data.
**Application mnesia started on node ejabberd@localhost**

How should i mention connection details of mysql?

I have refered this link:
https://docs.ejabberd.im/admin/databases/mysql/#adding-mysql-connection-configuration-to-ejabberd-config-file
